src/blocks/Text/DefaultTextBlockEditor.jsx

```diff
@@ -1,44 +1,44 @@
-import ReactDOM from 'react-dom';
-import React from 'react';
-import { readAsDataURL } from 'promise-file-reader';
-import Dropzone from 'react-dropzone';
-import { defineMessages, useIntl } from 'react-intl';
-import { useInView } from 'react-intersection-observer';
-import { Dimmer, Loader, Message, Segment } from 'semantic-ui-react';
+import ReactDOM from "react-dom";
```

Comment:
@sneridagh What's with the double-quotes? Jenkins is complaining about them.

Reply:
I saw, I just pushed... I don't know why my VSCode changed the formatting :(

Reply:
Maybe because I have it in a bare repository (no Volto around)...